relay-server/src/actors/processor.rs

```diff
@@ -59,13 +59,45 @@ use crate::envelope::{AttachmentType, ContentType, Envelope, Item, ItemType};
 use crate::extractors::RequestMeta;
 use crate::metrics_extraction::sessions::extract_session_metrics;
 use crate::metrics_extraction::transactions::extract_transaction_metrics;
-use crate::metrics_extraction::transactions::types::ExtractMetricsError;
+use crate::metrics_extraction::transactions::types::CommonTags;
+use crate::metrics_extraction::transactions::types::{
+    ExtractMetricsError, TransactionMeasurementTags, TransactionMetric,
+};
+use crate::metrics_extraction::IntoMetric;
 use crate::statsd::{RelayCounters, RelayHistograms, RelayTimers};
 use crate::utils::{
     self, get_sampling_key, log_transaction_name_metrics, ChunkedFormDataAggregator, FormDataIter,
     ItemAction, ManagedEnvelope, SamplingResult,
 };
 
+/// The length of a full measurement MRI, minus the name and the unit. This length is the same
+/// for every measurement-mri.
+///
+/// In 'fn process_measurements' we want to check if a measurement name is too long, to do that we
+/// check with a maximum length provided by the config, but that length represents the entire MRI name,
+/// and in process_measurements we only have access to the name part, and the unit part, so here
+/// we calculate the length of the remainder part. which isn't changing so we only need to do it once.
```

Comment:
This might help to understand what's going on:

```suggestion
/// we calculate the length of the remainder part. which isn't changing so we only need to do it once:
///
///     len("d:transactions/measurements.foobar@none") - len("foobar") - len("none") == len("d:transactions/measurements.@")
```